### DISCUSSION TITLE
Can‘t input anything in the inputbox called by Alt + /

### DISCUSSION THREAD

**Discussion by Marsirrr** [community/community#77832] [Copilot Conversations]
### Select Topic Area

Bug

### Body

![image](https://github.com/community/community/assets/26656034/21a91a93-2c91-4d07-b7eb-1b0c3a60963d)
The cursor is always outside the input box，copilot chat version: 0.1.1982.7773  , visual studio 2022 ver: 17.7.6

**Comment by Akash1134** [comment]
Hi @Marsirrr ,

Does this happen if you click within the "Ask Copilot" section and then input a comment? 

Thanks!

**Reply by Marsirrr** [reply]
It's ok when click within the "Ask Copilot".Doesn't work within the "Alt + /".I remember that in the previous version, using alt+/ would identify which code segment was selected, but that is no longer the case now.

**Reply by rheapatel23** [reply]
Hi, sorry to hear you are experiencing this, can you send your logs so our team can investigate this issue further? 

**How to provide logs instructions:** 
1. The ComponentModelCache folder from %localappdata%\microsoft\visualstudio\17.0(your instance)\ComponentModelCache, which contains files such as Microsoft.VisualStudio.Default.err. 
2. The ExtensionMetadata.mpack file from %localappdata%\microsoft\visualstudio\17.0(your instance)\Extensions\ExtensionMetdata.mpack
run the tool at https://aka.ms/vscollect and collect the logs placed at %TEMP%\vslogs.zip and zip the entire contents of %temp%\vsgithubcopilotlogs 

Thanks, 
Rhea Patel - Product Manager GitHub Copilot Chat for Visual Studio

**Reply by Marsirrr** [reply]
Could you please provide me with your email address? I would like to send these logs via email.

**Reply by rheapatel23** [reply]
@Marsirrr my email is rhea.patel@microsoft.com. Thank you :)

**Reply by MarkKingston** [reply]
Is there any news on a fix for this issue?

**Comment by proficad** [comment]
I have the same problem.

**Comment by z-henry** [comment]
Same issue

**Reply by hanxu1210** [reply]
same. and the same issue for clicking "ask copilot" in context menu.

**Comment by moagonghak** [comment]
still same problem in community 2022 17.13.2 version.

**Reply by moagonghak** [reply]
I found a solution - Do not use Visual Assist. 
now it works

**Reply by MarkKingston** [reply]
I use Visual Assist, Germany keyboard layout, ask mapped to 'Alt + #' on 17.13.6.
Slightly different set-up, but 'Alt + #' and Right Click -> 'Ask Copilot' both function as expected.
